### PR TITLE
fix description of `tidb_enable_ddl`

### PR DESCRIPTION
**Pull request by CbcWestwolf**
### What is changed, added or deleted? (Required)

Close https://github.com/pingcap/docs-cn/issues/12981

修正 `tidb_enable_ddl` 的描述

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/41298

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
@CbcWestwolf Could you please invite a tech reviewer? Thanks!

File: system-variables.md [hunk 1]
```diff
@@ -1201,7 +1201,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 是否持久化到集群：否，仅作用于当前连接的 TiDB 实例
 - 默认值: `ON`
```
Comment (Oreoxmt) [inline system-variables.md:1202]:
```suggestion
- 默认值：`ON`
```

File: system-variables.md [hunk 2]
```diff
@@ -1201,7 +1201,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
 - 可选值：`OFF`，`ON`
-- 用于设置该 TiDB 服务器是否运行 DDL 语句。
+- 用于设置该 TiDB 服务器是否参与 DDL owner 的选举。若当前 TiDB 集群中只有一台 TiDB 实例，则不能禁止该实例参与 DDL owner 的选举。
```
Comment (Oreoxmt) [inline system-variables.md:1204]:
```suggestion
- 用于设置该 TiDB 服务器是否参与 DDL owner 的选举。若当前 TiDB 集群中只有一台 TiDB 实例，则不能禁止该实例参与 DDL owner 的选举，即不能设置为 `OFF`。
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by xiongjiwei — APPROVED** [review]

File: system-variables.md [hunk 3]
```diff
@@ -1199,9 +1199,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 - 作用域：GLOBAL
 - 是否持久化到集群：否，仅作用于当前连接的 TiDB 实例
-- 默认值: `ON`
+- 默认值：`ON`
 - 可选值：`OFF`，`ON`
-- 用于设置该 TiDB 服务器是否运行 DDL 语句。
+- 用于设置该 TiDB 服务器是否参与 DDL owner 的选举。若当前 TiDB 集群中只有一台 TiDB 实例，则不能禁止该实例参与 DDL owner 的选举，即不能设置为 `OFF`。
```
Comment (xiongjiwei) [inline system-variables.md:1204]:
```suggestion
- 用于设置该 TiDB 实例是否可以成为 DDL owner。若当前 TiDB 集群中只有一台 TiDB 实例，则不能禁止该实例成为 DDL owner，即不能设置为 `OFF`。
```

File: tidb-configuration-file.md [hunk 4]
```diff
@@ -810,7 +810,7 @@ TiDB 服务状态相关配置。
 
 ### `tidb_enable_ddl`
 
-+ 用于表示该 tidb-server 是否运行 DDL 语句。
++ 用于表示该 tidb-server 是否参与 DDL owner 的选举。
```
Comment (Oreoxmt) [inline tidb-configuration-file.md:813]:
```suggestion
+ 用于表示该 tidb-server 是否可以成为 DDL owner。
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**